cristin-person/src/main/java/no/unit/nva/cristin/person/handler/PersonFetchHandler.java

```diff
@@ -50,14 +50,15 @@ protected Integer getSuccessStatusCode(Void input, Person output) {
 
     private void validateQueryParameters(RequestInfo requestInfo) throws BadRequestException {
         if (!requestInfo.getQueryParameters().keySet().isEmpty()) {
-            throw new BadRequestException(ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_LOOKUP);
+            throw new BadRequestException(ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_PERSON_LOOKUP);
         }
     }
 
     private String getValidId(RequestInfo requestInfo) throws BadRequestException {
-        attempt(() -> Integer.parseInt(requestInfo.getPathParameter(ID)))
-            .orElseThrow(failure -> new BadRequestException(ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_ID));
-
-        return requestInfo.getPathParameter(ID);
+        String identifier = requestInfo.getPathParameter(ID);
+        if (!Utils.isPositiveInteger(identifier)) {
```

Comment:
Here, the usage is the exact opposite of everywhere else in the project. I don't disagree that throwing the exception should be the exception, though. I'd refactor the method to Utils:isNegativeInteger and the rest of the code in the project to match this pattern in another PR.